olive/engine/packaging/packaging_generator.py

```diff
@@ -262,24 +262,20 @@ def _package_onnxruntime_packages(tempdir, pf_footprint: Footprint):
                 )
             run_subprocess(download_command)
 
-        # Download CPP onnxruntime package
-        cpp_ort_download_path = tempdir / "ONNXRuntimePackages" / "cpp"
-        cpp_ort_download_path.mkdir(parents=True, exist_ok=True)
-        if should_package_ort_cpu:
-            cpp_download_path = str(cpp_ort_download_path / f"microsoft.ml.onnxruntime.{ort_version}.nupkg")
-            _download_c_packages(True, is_nightly, ort_version, cpp_download_path)
-        if should_package_ort_gpu:
-            cpp_download_path = str(cpp_ort_download_path / f"microsoft.ml.onnxruntime.gpu.{ort_version}.nupkg")
-            _download_c_packages(False, is_nightly, ort_version, cpp_download_path)
-
-        # Download CS onnxruntime package
-        cs_ort_download_path = tempdir / "ONNXRuntimePackages" / "cs"
-        cs_ort_download_path.mkdir(parents=True, exist_ok=True)
-        if should_package_ort_cpu:
-            cs_download_path = str(cs_ort_download_path / f"microsoft.ml.onnxruntime.{ort_version}.nupkg")
-            _download_c_packages(True, is_nightly, ort_version, cs_download_path)
-        if should_package_ort_gpu:
-            _download_c_packages(False, is_nightly, ort_version, cs_download_path)
+        # Download CPP && CS onnxruntime package
+        lang_list = ["cpp", "cs"]
+        for language in lang_list:
+            ort_download_path = tempdir / "ONNXRuntimePackages" / language
+            ort_download_path.mkdir(parents=True, exist_ok=True)
+            if is_nightly:
+                _skip_download_c_package(ort_download_path)
+            else:
+                if should_package_ort_cpu:
```

Comment:
>should_package_ort_cpu [](http://example.com/codeflow?start=19&length=22)

should we download DML/OpenVino EP?

Reply:
Good call. I'll update this with dml and openvino.